metabase/metabase
# Move events between timelines

Epic https://github.com/metabase/metabase/issues/20289

Changes:
- Adds a modal to move events between timelines
- Refactors modals to remove code duplication between questions and collections modals

How to test:
- Create multiple timelines with events
- Go to a timeline, open menu for an event and click `Move event`
- Select another timeline and click `Move`
- Make sure the event is moved to another timeline

<img width="794" alt="Screenshot 2022-04-11 at 10 32 03" src="https://user-images.githubusercontent.com/8542534/162687054-87c8c2b4-77e8-40ea-a741-711edf920428.png">
<img width="652" alt="Screenshot 2022-04-11 at 10 32 46" src="https://user-images.githubusercontent.com/8542534/162687063-2658626f-1939-45d4-a9dd-c51341b34ff4.png">
<img width="657" alt="Screenshot 2022-04-11 at 10 32 54" src="https://user-images.githubusercontent.com/8542534/162687068-5f7220e8-1e58-442d-9c5f-cae3ab6275b0.png">
